src/components/memorygame/MemoryGameFlippableCard/index.css

```diff
@@ -0,0 +1,18 @@
+.memoryGameCardsContainer___card {
+  cursor: pointer;
+  height: clamp(20vh, 280px, 35vh);
+  width: 100%;
+  border-radius: 20px;
+  transition: all, 0.5s;
+}
+
+.memoryGameCardsContainer___card___image {
```

Comment:
imo something in its own component shouldn't have to know that it belongs to memoryGameCardsContainer. that violates the whole point of the component being independent from its parent components. just give it a css class that is scoped to the card

Reply:
nvm ignore this comment. I thought that memorygamecardscontainer was the parent css class but I believe it's only used here, which is good